_dashboards/workspace/workspace.md

```diff
@@ -0,0 +1,118 @@
+
+---
+
+#### Example Workspace object
+
+The following object shows a typical Workspace configuration:
+
+```typescript
+{
+  id: "M5NqCu",
+  name: "Analytics team",
+  description: "Analytics team workspace",
+  features: ["use-case-analytics"],
+}
+```
+{% include copy-curl.html %}
+
+The configuration creates an analytics team created with the `use-case-observability` feature set. Use cases map to specific OpenSearch Dashboards feature groups, limiting functionality to the defined set within each workspace. 
+
+Predefined use case options:
+
+- `use-case-observability`
+- `use-case-security-analytics`
+- `use-case-search`
+- `use-case-essentials`
+- `use-case-all`
+
+---
+
+## Associating saved objects with workspaces
+
+Saved objects in OpenSearch Dashboards, such as dashboards, visualizations, and index patterns, can be associated with specifif workspaces, improviing organization and accessibility as the volume of objects grows.
+
+The `workspaces` attribute, an array of strings, is added to saved objects to be linked with one or more workspaces. As a result, saved objects such as dashboards and visualizations are only accessible within their designated workspaces. 
+
+The following saved object shows a dashboard object associated with the workspace by `M5NqCu`:
+
+```typescript
+{
+  type: "dashboard",
+  id: "da123f20-6680-11ee-93fa-df944ec23359",
+  workspaces: ["M5NqCu"]
+}
+```
+{% include copy-curl.html %}
+
+Saved objects support association with multiple workspaces, facilitating cross-team collaboration and resource sharing. This feature is useful when an object is relevant to various team, projects, or use cases. 
+
+The following example shows a data source object linked to multiple workspaces:
+
+```typescript
+{
+  type: "data-source",
+  id: "da123f20-6680-11ee-93fa-df944ec23359",
+  workspaces: ["M5NqCu", "<TeamA-workspace-id>", "<Analytics-workspace-id>"]
+}
+```
+{% include copy-curl.html %}
+
+## Non-workspace saved objects
+
+Not all saved objects in OpenSearch Dashboards are workspace-associated. Some objects operate independently of workspace framework. These objects lack `workspace` attribute and serve system-wide functions. For example, the global user interface settings object manages configurations affecting the entire OpenSearch Dashboards interface to maintain consistent functionality across all workspaces.
+
+This dual approach, workspace-associated and independent objects, allows OpenSearch Dashboards to balance granular, context-specific customization with overall system consistency. 
+
+## Enabling the Workspace feature
+
+In your `opensearch_dashboards.yml` file, set the following option:
+
+```yaml
+workspace.enabled: true
+uiSettings:
+  overrides:
+    "home:useNewHomePage": true
+```
+{% include copy-curl.html %}
+
+If your cluster has the security plugin installed, then multitenancy must be disabled to avoid conflicts with workspaces of similar functionality.
```

Comment:
```suggestion
If your cluster has the Security plugin installed, then multi-tenancy must be disabled to avoid conflicts with workspaces providing similar functionality:
```